docs/api/tutorials/assertions.md

```diff
@@ -71,10 +71,108 @@ This API will return a unique identifier (URN) for the new assertion if you were
 }
 ```
 
+</TabItem>
+<TabItem value="python" label="Python">
+
+```python
```

Comment:
should we move these embedded scripts to be inlined?

```
{{ inline /metadata-ingestion/examples/library/create_freshness_assertion.py show_path_as_comment }}
```

I haven't checked but I guess inlined scripts are validated (linted, tested?) so less risk to become out-dated